article/conf/routes

```diff
@@ -5,9 +5,9 @@
 # For dev machines
 GET     /assets/*path               dev.DevAssetsController.at(path)
 
-GET     /_healthcheck               conf.HealthCheck.healthcheck()
+GET     /_healthcheck               conf.HealthCheck.healthCheck()
 
-GET     /_cdn_healthcheck           controllers.CdnHealthcheckController.healthcheck()
+GET     /_cdn_healthcheck           conf.HealthCheck.healthCheck()
```

Comment:
might be worth putting that comment in the file itself as if someone else comes across this file they won't find this PR